stratum/hal/lib/barefoot/BUILD

```diff
@@ -145,6 +145,10 @@ stratum_cc_library(
     name = "bf_sde_wrapper",
     srcs = ["bf_sde_wrapper.cc"],
     hdrs = ["bf_sde_wrapper.h"],
+    defines = select({
+        "//stratum/hal/lib/pi:sde_9_4_0": ["SDE_9_4_0"],
+        "//conditions:default": [],
```

Comment:
We should probably list all versions here and hard fail if the `--define` flags was not set.